code/game/machinery/bots/bots.dm

```diff
@@ -84,9 +84,9 @@
 		if(hasvar(W,"force") && hasvar(W,"damtype"))
 			switch(W.damtype)
 				if("fire")
-					src.health -= W.force * fire_dam_coeff
+					health -= W.force * W.demolition_mod * fire_dam_coeff
 				if("brute")
-					src.health -= W.force * brute_dam_coeff
+					health -= W.force * W.demolition_mod * brute_dam_coeff
```

Comment:
There's a few extra things that could have the `demolition_mod` modifier added to them.
Specifically:
[/obj/structure/displaycase](https://github.com/cmss13-devs/cmss13/blob/master/code/game/objects/structures/displaycase.dm#L60)
[/obj/structure/mineral_door](https://github.com/cmss13-devs/cmss13/blob/b5e180c53acbca0fcc195e9b9d13a7c42b3f4a92/code/game/objects/structures/mineral_doors.dm#L105)
[/obj/structure/alien/movable_wall](https://github.com/cmss13-devs/cmss13/blob/b5e180c53acbca0fcc195e9b9d13a7c42b3f4a92/code/game/turfs/walls/wall_types.dm#L1047) (could skip this one actually, I don't think it's used anywhere)
[/turf/closed/wall/resin](https://github.com/cmss13-devs/cmss13/blob/b5e180c53acbca0fcc195e9b9d13a7c42b3f4a92/code/game/turfs/walls/wall_types.dm#L1283)
[/obj/structure/mineral_door/resin](https://github.com/cmss13-devs/cmss13/blob/b5e180c53acbca0fcc195e9b9d13a7c42b3f4a92/code/modules/cm_aliens/XenoStructures.dm#L394)